api/src/services/survey-critter-service.ts

```diff
@@ -26,17 +26,28 @@ export class SurveyCritterService extends DBService {
    * @param {string} critterId
    * @returns {*}
    */
-  async addCritterToSurvey(surveyId: number, critterBaseCritterId: string): Promise<number> {
-    return this.critterRepository.addCritterToSurvey(surveyId, critterBaseCritterId);
+  async addCritterToSurvey(surveyId: number, critterBaseCritterId: string): Promise<void> {
+    this.critterRepository.addCritterToSurvey(surveyId, critterBaseCritterId);
```

Comment:
Either `return` this Promise, or `await` it. In its current state, this isn't really an async method because it dispatches `critterRepository.addCritterToSurvey` and then immediately returns void.

Reply:
I would prefer using `return` here, but either will achieve the same result